Library/Formula/lcab.rb

```diff
@@ -0,0 +1,21 @@
+class Lcab < Formula
+  homepage "http://ohnopub.net/~ohnobinki/lcab/"
+  url "ftp://mirror.ohnopub.net/mirror/lcab-1.0b12.tar.gz"
```

Comment:
Ah yes, this `b` looks like it’s a beta version but I can’t find any other download.